### ISSUE TITLE
kbd67/mkiirgb/v3 fails to compile - quantum/rgb_matrix.c operator has no left operand

### ISSUE THREAD

**Issue by ginzberg** [qmk/qmk_firmware#13965]
## Describe the Bug
The `kbd67/mkiirgb/v3` seems to fail to compile, regardless of keymap (attempted both `via` and `default`).

On my system, I am able to compile firmwares (successfully built `nk65` and `mkiirgb/v2` with no errors)

Console Output:
```qmk compile -kb kbdfans/kbd67/mkiirgb/v3 -km default
Ψ Compiling keymap with make -j 1 kbdfans/kbd67/mkiirgb/v3:default


QMK Firmware 0.13.34
Making kbdfans/kbd67/mkiirgb/v3 with keymap default

avr-gcc (GCC) 11.2.0
Copyright (C) 2021 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

Compiling: quantum/rgb_matrix.c                                                                    quantum/rgb_matrix.c:70:36: error: operator '!=' has no left operand
   70 | #if RGB_DISABLE_WHEN_USB_SUSPENDED != 1
      |                                    ^~
 [ERRORS]
 |
 |
 |
make[1]: *** [tmk_core/rules.mk:436: .build/obj_kbdfans_kbd67_mkiirgb_v3_default/quantum/rgb_matrix.o] Error 1
Make finished with errors
make: *** [Makefile:539: kbdfans/kbd67/mkiirgb/v3:default] Error 1 
```

## System Information

 - Keyboard:  `kbd67/mkiirgb/v3`
   - Revision (if applicable):  `v3`
 - Operating system: Arch Linux x64 (kernel=`5.13.6-zen1-1-zen`)
 - AVR GCC version: `11.2`
 - ARM GCC version: `11.2`
 - QMK Firmware version: `0.13.34`
 - Any keyboard related software installed? 
   - [ ] AutoHotKey
   - [ ] Karabiner
   - [ ] Other:

## Additional Context
I have zero experience with QMK. I saw that this v3 revision of the kbd67 just made it to QMK, and I was interested in building a 1000hz firmware. Very likely I'm doing something wrong, but the fact I was able to build other firmwares for other keyboards prompted me to send this issue.

**Comment by ginzberg** [comment]
Update:

I just read a comment from Dasky on discord, and this resolves the issue....

Change the line in `kbdfans/kbd67/mkiirgb/v3/config.h` that reads:
`#define RGB_DISABLE_WHEN_USB_SUSPENDED // turn off effects when suspended`
to 
`#define RGB_DISABLE_WHEN_USB_SUSPENDED true // turn off effects when suspended`

**Comment by vinorodrigues** [comment]
This is fixed with pull request #13978

**Comment by vinorodrigues** [comment]
This seems to be fixed now with pull request #14084
